js/skin.js

```diff
@@ -165,17 +171,19 @@ var Skin = React.createClass({
   /**
    * @public
    * @description set accessibilityControlsEnabled to true for focus event
+   * @return {undefined}
```

Comment:
We should take this out since it doesn't return anything.

Reply:
\+